site/docs/ci/example-github-actions.md

```diff
@@ -5,7 +5,9 @@ title: "Example: Github Actions"
 
 # Github Actions
 
-In this section we'll show an example of how to run Preevy in GitHub Actions pipeline.
+In this section we'll show an example of how to run Preevy using our GitHub Actions [preevy-up](https://github.com/marketplace/actions/preevy-up) and [preevy-down](https://github.com/marketplace/actions/preevy-down).
```

Comment:
probably need to explain it's using AWS lightsail and S3 specifically (and that is the reason for the authentication)